src/Bundle/CustomReport/Hydrator/ColumnHydrator.php

```diff
@@ -34,10 +41,13 @@ public function hydrateColumnInfo(ColumnInformation $information): CustomReportC
         );
     }
 
-    public function getCustomReportColumnConfiguration(array $columns): array
+    public function getCustomReportColumnConfiguration(Config $report): array
     {
         $columnConfig = [];
-        foreach ($columns as $column) {
+        $metadataMap = $this->getMetadataMap($report);
+        foreach ($report->getColumnConfiguration() as $column) {
+            // @var ColumnInformation|null $metadata
```

Comment:
Let's keep it consistent with /** */ comment? 
```suggestion
            /** @var ColumnInformation|null $metadata */
```

Reply:
thanks :)